debian/control

```diff
@@ -1,61 +1,61 @@
  bison, flex, docbook-to-man, help2man, xsltproc, doxygen,
  dh-python,
- g++ (>= 7),
- python3, python3-all-dev (>= 3.1)
+ g++ (>= 4:5-0),
+ python3, python3-all-dev (>= 3.1), python3-numpy
 Build-Depends-Indep: graphviz
 Build-Conflicts: libopenmpi-dev (= 1.3.2-2)
 X-Python3-Version: >= 3.6
 Vcs-Browser: https://salsa.debian.org/debian/boost
 Vcs-Git: https://salsa.debian.org/debian/boost.git
-Standards-Version: 4.1.1
+Standards-Version: 4.6.2
 
-Package: ceph-libboost1.79-dev
+Package: ceph-libboost1.82-dev
 Architecture: any
 Multi-Arch: same
 Section: libdevel
-Depends: ${misc:Depends}, ${shlibs:Depends}, libstdc++-4.8-dev | libstdc++-dev
-Suggests: ceph-libboost1.79-doc,
- ceph-libboost-atomic1.79-dev,
- ceph-libboost-chrono1.79-dev,
- ceph-libboost-container1.79-dev,
- ceph-libboost-context1.79-dev [any-i386 any-amd64 armel armhf arm64 mips mipsel powerpc ppc64el],
- ceph-libboost-coroutine1.79-dev [any-i386 any-amd64 armel armhf arm64 mips mipsel powerpc ppc64el],
- ceph-libboost-date-time1.79-dev,
- ceph-libboost-exception1.79-dev,
- ceph-libboost-fiber1.79-dev [any-i386 any-amd64 armel armhf arm64 mips mipsel powerpc ppc64el],
- ceph-libboost-filesystem1.79-dev,
- ceph-libboost-graph1.79-dev,
- ceph-libboost-graph-parallel1.79-dev,
- ceph-libboost-iostreams1.79-dev,
- ceph-libboost-locale1.79-dev,
- ceph-libboost-log1.79-dev,
- ceph-libboost-math1.79-dev,
- ceph-libboost-program-options1.79-dev,
- ceph-libboost-python1.79-dev,
- ceph-libboost-random1.79-dev,
- ceph-libboost-regex1.79-dev,
- ceph-libboost-serialization1.79-dev,
- ceph-libboost-stacktrace1.79-dev,
- ceph-libboost-system1.79-dev,
- ceph-libboost-test1.79-dev,
- ceph-libboost-thread1.79-dev,
- ceph-libboost-timer1.79-dev,
- ceph-libboost-type-erasure1.79-dev,
- ceph-libboost-wave1.79-dev,
- ceph-libboost1.79-tools-dev,
- libmpfrc++-dev, libntl-dev
+Depends: ${misc:Depends}, ${shlibs:Depends}, libstdc++-${gxx:major}-dev
+Suggests: ceph-libboost1.82-doc,
+ ceph-libboost-atomic1.82-dev,
+ ceph-libboost-chrono1.82-dev,
+ ceph-libboost-container1.82-dev,
+ ceph-libboost-context1.82-dev [i386 hurd-i386 kfreebsd-i386 amd64 kfreebsd-amd64 armel armhf arm64 mips mipsel mips64el powerpc ppc64el riscv64 s390x],
+ ceph-libboost-contract1.82-dev,
+ ceph-libboost-coroutine1.82-dev [i386 hurd-i386 kfreebsd-i386 amd64 kfreebsd-amd64 armel armhf arm64 mips mipsel mips64el powerpc ppc64el riscv64 s390x],
+ ceph-libboost-date-time1.82-dev,
+ ceph-libboost-exception1.82-dev,
+ ceph-libboost-fiber1.82-dev [i386 hurd-i386 kfreebsd-i386 amd64 kfreebsd-amd64 armel armhf arm64 mips mipsel mips64el powerpc ppc64el riscv64 s390x],
+ ceph-libboost-filesystem1.82-dev,
+ ceph-libboost-graph1.82-dev,
+ ceph-libboost-graph-parallel1.82-dev,
+ ceph-libboost-iostreams1.82-dev,
+ ceph-libboost-locale1.82-dev,
+ ceph-libboost-log1.82-dev,
+ ceph-libboost-math1.82-dev,
+ ceph-libboost-mpi1.82-dev,
+ ceph-libboost-mpi-python1.82-dev,
+ ceph-libboost-numpy1.82-dev,
```

Comment:
please note, this brings back mpi and numpy librareis. which was dropped in 9ba98658561c2538ae33aba223785f253b0c9f61 and b93d32496cd2ab4f75cdfab6361514aad9d61466. if the build environment has been changed since then, i am fine with this change. the extra build dependencies and build time should not be taken into consideration. as the ceph-boost libraries build is not a recurring task with high frequency.

Reply:
Thank you. I didn't realize they'd been dropped. I just built it on a Jammy container on my own system. I'll pull them back out again.

Reply:
@adamemerson i see. i just checked https://jenkins.ceph.com/job/ceph-pull-requests/115386/consoleFull as a sample of what distro we use for performing "make check" run. i found
```
+ '[' -r /etc/os-release ']'
+ source /etc/os-release
++ PRETTY_NAME='Ubuntu 22.04.1 LTS'
++ NAME=Ubuntu
++ VERSION_ID=22.04
++ VERSION='22.04.1 LTS (Jammy Jellyfish)'
++ VERSION_CODENAME=jammy
++ ID=ubuntu
++ ID_LIKE=debian
```

as i think it is the only consumer of these ceph-boost packages. so, we are using xenial as back then, we were suffering from the slow build and "make check" cycles. that's why we decided to create the prebuilt boost packages to speed up the bulid. so now that we are using jammy, i think we are safe in this regard. to drop them was a last resort back then. as it could make our lives harder when cherry-picking upstream changes . so i'd prefer align with upstream as much as possible. so no need to put them back.

Reply:
All right, thank you.